### PR TITLE
Initial Launch

### PR DESCRIPTION
**Pull request by daronyondem**

### CODE REVIEW

File: Hands-on lab/Manual-resource-setup.md [hunk 1]
```diff
@@ -0,0 +1,508 @@
...
+     - **Subscription (1)**: Select the subscription you are using for this hands-on lab.
+     - **Resource Group (2)**: Select the hands-on-lab-SUFFIX resource group from the list of existing resource groups.
+
+   - Instance Details:
+
+     - **Account Name (3)**: Enter contoso-cdb-SUFFIX.
+     - **Location (4)**: Select the location you are using for resources in this hands-on lab.
+     - **Capacity Mode (5)**: Choose **Serverless**.
+
+   ![On the Create Azure Cosmos DB Account blade, the values specified above are entered into the appropriate fields.](media/cosmos-db-create-resource.png "Create new Azure Cosmos DB Account")
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/Manual-resource-setup.md:410]: Check the image link please.  Image not resolving when I view the document.  Thanks!

image

Comment (daronyondem) [reply Hands-on lab/Manual-resource-setup.md]: Fixed. https://github.com/microsoft/MCW-AI-led-business-process-automation/pull/1/commits/6270a46805b5dcaf0cec20a2745af16e468a5b1e

**Comment by DawnmarieDesJardins** [comment]
QC pass - PPT
We're missing sections of the PPT.  The PPT should copy the trainer guide in content, only with talking points on the slides, full answers in the Notes for trainers.

No common scenario slide.  
Missing design a solution slides (what questions students are supposed to answer)
Need preferred solutions slides that answer the design a solution questions.

**Comment by daronyondem** [comment]
Hi @DawnmarieDesJardins 

I'm confused. Are we looking at the same PPT? 

- The only missing piece I can see is the **Common Scenario**. 
- The preferred solution slide is #13
- I'm not sure what design solution questions refer to. @DawnmarieDesJardins Can you share an example? @codingbandit any ideas? I see customer needs #7, objections #8, and objection handling #14 slides if those are the ones referred to as "what questions students are supposed to answer"

**Comment by DawnmarieDesJardins** [comment]
Thanks @daronyondem - I answered via email so I could try to show you.

**Pull request merged**